extensions/wikia/DesignSystem/bower_components/design-system/dist/scss/wds-components/_banner-notification.scss

```diff
@@ -2,6 +2,11 @@
 	background-color: $wds-color-white;
 	color: $wds-color-light-gray;
 	display: flex;
+	transition: opacity 0.4s;
+
+	&--transparent {
```

Comment:
we do not use BEM modifiers in DS. I'd change it to `&.wds-is-transparent` or `&.wds-is-hidden`

Reply:
See https://github.com/Wikia/design-system/pull/169#discussion_r142486046